packages/rpc/src/common.ts

```diff
@@ -96,17 +98,34 @@ export type RpcRequest = WithHash<{
     amount: `0x${string}`,
     additionalOrders?: Nullish<AdditionalOrder[]>,
   ];
-  cf_swap_rate_v3: [
-    fromAsset: UncheckedAssetAndChain,
-    toAsset: UncheckedAssetAndChain,
-    amount: `0x${string}`,
-    brokerComission?: Nullish<number>,
-    dcaParams?: Nullish<{
-      number_of_chunks: number;
-      chunk_interval: number;
-    }>,
-    additionalOrders?: Nullish<AdditionalOrder[]>,
-  ];
+  cf_swap_rate_v3:
+    | [
+        fromAsset: UncheckedAssetAndChain,
+        toAsset: UncheckedAssetAndChain,
+        amount: `0x${string}`,
+        brokerComission?: Nullish<number>,
+        dcaParams?: Nullish<{
+          number_of_chunks: number;
+          chunk_interval: number;
+        }>,
+        additionalOrders?: Nullish<AdditionalOrder[]>,
+      ]
+    | [
+        fromAsset: UncheckedAssetAndChain,
+        toAsset: UncheckedAssetAndChain,
+        amount: `0x${string}`,
+        brokerComission: number,
+        dcaParams?: Nullish<{
+          number_of_chunks: number;
+          chunk_interval: number;
+        }>,
+        ccm_data?: Nullish<{
+          gas_budget: HexString;
+          message_length: number;
+        }>,
+        exclude_fees?: Nullish<SwapFeeType[]>,
```

Comment:
small nit because we use camelcase for the other params
```suggestion
        ccmData?: Nullish<{
          gas_budget: HexString;
          message_length: number;
        }>,
        excludeFees?: Nullish<SwapFeeType[]>,
```

Reply:
Right, thaanks ☺️